users/art/art.c

```diff
@@ -1,103 +1,114 @@
 #include "art.h"
 #include "string.h"
+#include "ctype.h"
+#include "secret_definitions.h"
+#include "funcs/led_funcs.c"
```

Comment:
Shouldn't include c files.  They should be SRC'ed

Reply:
should be done now